src/components/organisms/ContentBox2.jsx

```diff
@@ -0,0 +1,458 @@
+         white-space: nowrap;
+       }
+     `}
+  }
+
+`
+
+/*
+ * Defined content padding in alignment with links
+ */
+
+const PositionLeft = ['BOTTOM_LEFT', 'CENTER_LEFT', 'TOP_LEFT']
+const PositionRight = ['BOTTOM_RIGHT', 'CENTER_RIGHT', 'TOP_RIGHT']
+const PositionBottom = ['BOTTOM_LEFT', 'BOTTOM_CENTER', 'BOTTOM_RIGHT']
+const PositionBottomLeft = ['BOTTOM_LEFT', 'BOTTOM_CENTER']
+const PositionBottomRight = ['BOTTOM_RIGHT', 'BOTTOM_CENTER']
+
+const defineContentPadding = ({ isPreviewMobile, isSidePreview }) => ({
+  left: `padding-left:  ${(isSidePreview && 3.3) || (isPreviewMobile && 4.5) || 6}rem`,
+  right: `padding-right: ${(isSidePreview && 3.3) || (isPreviewMobile && 4.5) || 6}rem`,
+  bottom: `padding-bottom: ${(isSidePreview && 3.3) || (isPreviewMobile && 4.5) || 6}rem`,
+  none: 'padding: 0',
+})
+
+const getContentPadding = ({ content, links, isPreviewMobile, isSidePreview }) => {
+  const contentPositionDesktop = content.positionDesktop || content.position
+  const contentPositionMobile = (content.isDifferentPositions && content.positionMobile) || content
+
+  const contentPadding = defineContentPadding({ isPreviewMobile, isSidePreview })
+
+  const linkPosition = links.list.length > 0 ? links.position : ''
+
+  const paddingIndexDesktop =
+    (linkPosition === 'CENTER_LEFT' && PositionLeft.includes(contentPositionDesktop) && 'left') ||
+    (linkPosition === 'CENTER_RIGHT' &&
+      PositionRight.includes(contentPositionDesktop) &&
+      'right') ||
+    (PositionBottomLeft.includes(linkPosition) &&
+      PositionBottomLeft.includes(contentPositionDesktop) &&
+      'bottom') ||
+    (PositionBottomRight.includes(linkPosition) &&
+      PositionBottomRight.includes(contentPositionDesktop) &&
+      'bottom') ||
+    'none'
+
+  const paddingIndexMobile =
+    // (linkPosition === 'CENTER_LEFT' && PositionLeft.includes(logoPositionMobile) && 'left') ||
+    // (linkPosition === 'CENTER_RIGHT' && PositionRight.includes(logoPositionMobile) && 'right') ||
+    //(PositionBottom.includes(linkPosition) &&
+    (linkPosition > '' && PositionBottom.includes(contentPositionMobile) && 'bottom') ||
+    // (isPreviewMobile && 'bottom') ||
+    'none'
+
+  return {
+    desktop: contentPadding[paddingIndexDesktop],
+    mobile: contentPadding[paddingIndexMobile],
+  }
+}
+
+const getContentPosition = ({ content }) => {
+  const positionDesktop =
+    (content.positionDesktop > '' && content.positionDesktop) || content.position
+
+  const classnameDesktop =
+    (positionDesktop > '' && positionDesktop.toLowerCase().replace('_', '-')) || 'top-center'
+
+  const contentPosition =
+    (content.isDifferentPositions && content.positionMobile) ||
+    content.positionDesktop ||
+    content.position
+
+  const classnameMobile =
+    (contentPosition > '' && contentPosition.toLowerCase().replace('_', '-')) || 'top-center'
+
+  return {
+    classnameMobile,
+    classnameDesktop,
+  }
+}
+
+export const ContentBox = ({
```

Comment:
@Jonathansoufer could you explain the difference between **ContentBox.jsx** and **ContentBox2.jsx** ?